package-lock.json

```diff
@@ -1,8595 +1,8 @@
 {
   "name": "@toznysecure/sdk",
   "version": "1.3.6",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
as @house9 alludes to, in npm v7 they changed how lockfiles are structured.

can you regenerate this file with an upgraded `npm` installed to prevent downgrading?
```sh
# upgrade npm
npm upgrade -g npm@latest
# regenerate package-lock
npm i
```

Reply:
(insert a @galxy25 rant on the benefits of containerization 😆) 
(though to be fair, one is rarely using `npm` to add new packages from the container :trollface:)